server_plugin/lib/src/completion.dart

```
@@ -890,6 +898,28 @@ class TemplateCompleter {
     }
   }
 
+  void suggestRefValues(
+      List<DirectiveBinding> directives, CompletionCollector collector) {
+    // Keep map of all 'exportAs' name seen. Don't suggest same name twice.
```

Probably not - but currently there is not indicated from the Angular compiler when this occurs; it just silently fails. It would be nice if our plugin can at least provide a warning in such cases rather than leaving Angular developers scratching their heads in the rare cases where this does happen.